e2e/custom-test-sequencer/testSequencer.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const fs = require('fs');
```

Comment:
should use `import`

Reply:
Because it doesn't run with babel, using import will cause error.